### DISCUSSION TITLE
Add/Remove button for metadata dictionary

### DISCUSSION THREAD

**Discussion by 28106830** [godotengine/godot-proposals#7318] [Editor]
### Describe the project you are working on

In addition, the metadata dictionary does not have a deletion function. If you make a mistake, you can only delete all of them and rewrite them again

### Describe the problem or limitation you are having in your project

After adding metadata to the dictionary

If there is a problem with a certain item and it cannot be deleted, can I add a delete button键

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

NULL

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

NULL

### If this enhancement will not be used often, can it be worked around with a few lines of script?

NULL

### Is there a reason why this should be core and not an add-on in the asset library?

YES

**Comment by KoBeWi** [comment]
There is a delete button:
![image](https://github.com/godotengine/godot-proposals/assets/2223172/ff0ae480-c1e0-472c-b667-cb712c3009bb)

**Comment by 28106830** [comment]
您说的是对的，但这是删除整个元组



------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Bozhidar ***@***.***&gt;; 
发送时间: 2023年7月18日(星期二) 晚上7:44
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [godotengine/godot-proposals] Add/Remove button for metadata dictionary (Discussion #7318)






有一个删除按钮：


—
直接回复此邮件，在GitHub上查看，或者取消订阅.
***@***.***与&gt;.

**Comment by 28106830** [comment]
Only delete tuples, not individual ones




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "godotengine/godot-proposals"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年7月18日(星期二) 晚上7:44
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [godotengine/godot-proposals] Add/Remove button for metadata dictionary (Discussion #7318)





 
There is a delete button:

**Reply by timothyqiu** [reply]
If you want to remove an item from a dictionary, click the "pen" (edit) icon, and there's an option "Remove Item" at the end of the list.
